MAINTAINERS.yml

```diff
@@ -4173,6 +4173,18 @@ Panasonic Platforms:
   labels:
     - "platform: Panasonic"
 
+Toradex Platforms:
+  status: maintained
+  maintainers:
+    - drewmoseley
+  collaborators:
+    - GabrielHAFs
+    - hiagofranco
+  files:
+    - boards/arm/verdin_imx8m*/
+  labels:
+    - "platform: Toradex"
```

Comment:
to be removed, none of these people are zephyr collaborators

Reply:
I'm unclear; remove the whole addition to MAINTAINERS.yml or just the "collaborators" section?

Reply:
the whole addition